src/helm/proxy/services/server_service.py

```diff
@@ -58,7 +59,10 @@ def __init__(self, base_path: str = "prod_env", root_mode=False, mongo_uri: str
         self.toxicity_classifier_client: Optional[ToxicityClassifierClient] = None
 
     def get_general_info(self) -> GeneralInfo:
-        return GeneralInfo(version=VERSION, example_queries=example_queries, all_models=ALL_MODELS_METADATA)
+        # Can't send release_dates in ModelMetadata bacause dates cannot be round-tripped to and from JSON easily.
+        # TODO(#2158): Either fix this or delete get_general_info.
+        all_models = [dataclasses.replace(model_metadata, release_date=None) for model_metadata in ALL_MODELS_METADATA]
+        return GeneralInfo(version=VERSION, example_queries=example_queries, all_models=all_models)
```

Comment:
Actually a pre-existing issue that wasn't caught in the test before because the test forgot to register models, so `ALL_MODELS_METADATA` was empty.